routers/api/v1/org/member.go

```diff
@@ -18,15 +19,21 @@ import (
 
 // listMembers list an organization's members
 func listMembers(ctx *context.APIContext, publicOnly bool) {
-	var members []*models.User
-
-	members, _, err := models.FindOrgMembers(&models.FindOrgMembersOpts{
+	opts := &models.FindOrgMembersOpts{
 		OrgID:       ctx.Org.Organization.ID,
 		PublicOnly:  publicOnly,
 		ListOptions: utils.GetListOptions(ctx),
-	})
+	}
+
+	count, err := models.CountOrgMembers(opts)
+	if err != nil {
+		ctx.InternalServerError(err)
+		return
+	}
+
+	members, _, err := models.FindOrgMembers(opts)
```

Comment:
Can't we just get the count here?

```suggestion
	members, count, err := models.FindOrgMembers(opts)
```

Reply:
So looking at the code nope - the 2nd return of this function is a map[int64]bool{} saying whether the member is public or not.